### PR TITLE
Add commitchecker CI job example

### PR DESCRIPTION
**Pull request by jsafrane**
I need a dummy PR to trigger image build, so I am adding an example. I am not 100% sure it is a correct one, I'll fix it once I know how it works together.

### CODE REVIEW

File: commitchecker/README.md [hunk 1]
```diff
@@ -16,3 +16,44 @@ from https://github.com/openshift/kubernetes/tree/0abcd84431df81ed9f2a1846b5045e
...
+
+```yaml
+tests:
+...
+
+- as: verify-commits
+  commands: |
+    commitchecker --start ${PULL_BASE_SHA:-master}
+  container:
+    clone: true
```
Comment (ncdc) [inline commitchecker/README.md:57]: According to the docs for this field, it defaults to true if the `from` image is in `base_images` FYI

**Review by soltysh — APPROVED** [review]
/approve

File: commitchecker/README.md (hunk 1 above)
Comment (jsafrane) [reply commitchecker/README.md]: Indeed, I removed `clone: true` as unnecessary.

@ncdc PTAL, I copied the example from [the AWS EBS CSI driver jobs](https://github.com/openshift/release/pull/42462/files)

Comment (ncdc) [reply commitchecker/README.md]: 👍 yeah, we've turned these on for our openshift/operatator-framework-{catalogd, operator-controller, rukpak} repos. Thanks again for turning this into a reusable image!

File: commitchecker/README.md [hunk 2]
```diff
@@ -16,3 +16,46 @@ from https://github.com/openshift/kubernetes/tree/0abcd84431df81ed9f2a1846b5045e
...
+Note that the `commitchecker` image is available only with tags `4.14` and newer.
+
+2. Add `verify-commits` presubmit CI job that clones your repository, applies the PR-under-test and runs `commitchecker` image with the git clone inside:
+
+```yaml
+tests:
+...
+- as: verify-commits
+  commands: |
+    commitchecker --start ${PULL_BASE_SHA:-master}
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply commitchecker/README.md]: Fixed

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**